### DISCUSSION TITLE
Intellibrite Color Switching w/ 3HP Relay

### DISCUSSION THREAD

**Discussion by celestinjr** [tagyoureit/nodejs-poolController#944] [Q&A]
Currently in the process of planning my build, which will include repurposing of my AquaLogic AQL-PS-8 panel.  The AquaLogic panel has (8) 3HP relays available for reuse, for which I should only need (3) - Polaris booster pump and two blowers.

Would it be ill advised to power my (2) Intellibrite lights via a 3HP relay, getting a +24V control signal from the Pi Relay HAT?  I ask, since I'm not sure the larger 3HP relays will hold up to the cycling that it would see when setting the color scheme.

My only incentive to do this is to minimize the amount of 120V circuits in my controller enclosure.

Thanks!

**Comment by tagyoureit** [comment]
The intellibrite lights should be hooked up to a transformer which provides 12vac.  You want to turn on/off individual lights on the 12v side.  Don't power cycle the transformer to change lighting effects.  

![image](https://github.com/tagyoureit/nodejs-poolController/assets/1646840/46ca50a7-6dcc-4e41-a2d2-453e16741e26)

Even in the [wiki](https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#lighting-circuits) it shows switching before the transformer, but switching the lights on the 12vac side will give you control of the individual lights.

**Reply by celestinjr** [reply]
I should have clarified. I have the 120VAC version of the Intellibrite Architectural series. SN 602187.  They require switching 120VAC to select the mode.

**Reply by tagyoureit** [reply]
Gotcha... I'm not an expert but I don't think the 3HP relays are designed to handle the cycling.  You would be better off with 120v relays.

**Reply by celestinjr** [reply]
The Omron 3HP relays in my Aqualogic panel are rated for 25A @ 220VAC, but I agree they're unlikely to hold up with the switching frequency needed for the Intellibrites.

My current plan is to utilize this [solid state relay board](https://www.sainsmart.com/products/2-channel-5v-solid-state-relay), which is rated 0.1-2A @ 75-264VAC.  The 28W Intellibrites only draw ~0.25A @ 120VAC, so I believe this will give me a more reliable solution.
